### DISCUSSION TITLE
Elo estimates?

### DISCUSSION THREAD

**Discussion by carsonetb** [SebLague/Chess-Challenge#394] [General]
I was wondering what elo estimates people have for their bots, so that I could have some context for what I think mine is (Around 1400-1500).

**Comment by 0qln** [comment]
How did you measure your bot's elo? Is it just a guess or are there methods for doing this?

**Reply by carsonetb** [reply]
Kind of a guess, I've been playing against the chess.com bots wtih 5 seconds think time, it can sometimes beat the 1500 bot.

**Reply by 0qln** [reply]
I noticed after writing my comment, on the [Chess Bot Coding discord](https://github.com/SebLague/Chess-Challenge/discussions/156#discussion-5428295), is a guide on how to measure youre bots exact elo.
Although the elo is measured relative to other bots, and is not nesseceraly compatible with FIDE elo, it might be more useful in the scope of this challenge.

**Reply by carsonetb** [reply]
Oh, that's interesting.
